docs/reference/method/MongoDBClient-createClientEncryption.txt

```diff
@@ -0,0 +1,50 @@
+=========================================
+MongoDB\\Client::createClientEncryption()
+=========================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Definition
+----------
+
+.. phpmethod:: MongoDB\\Client::createClientEncryption()
+
+   Returns a ClientEncryption object for manual encryption and decryption of
+   values.
+
+   .. code-block:: php
+
+      function createClientEncryption(array $options): MongoDB\Driver\ClientEncryption
+
+   This method has the following parameters:
+
+   .. include:: /includes/apiargs/MongoDBClient-method-createClientEncryption-param.rst
+
+   The ``$options`` parameter supports all options documented in the extension
+   manual. For the ``keyVaultClient`` option, an instance of ``MongoDB\Client``
+   is automatically unwrapped and the ``MongoDB\Driver\Manager`` instance is
+   passed to the extension.
+
+Return Values
+-------------
+
+A MongoDB\Driver\ClientEncryption instance which can be used to encrypt and
```

Comment:
```
:phpclass:`MongoDB\\Driver\\ClientEncryption`
```

Reply:
Sorry, this should be `:php:` and you'll need to add `<class.mongodb-driver-clientencryption>` (preemptively).